x-pack/plugins/security_solution/server/lib/detection_engine/signals/threshold/build_threshold_aggregation.test.ts

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  buildThresholdMultiBucketAggregation,
+  buildThresholdSingleBucketAggregation,
+} from './build_threshold_aggregation';
+
+describe('build_threshold_aggregation', () => {
+  const timestampField = '@timestamp';
+
+  describe('build_threshold_multi_bucket_aggregation', () => {
+    const threshold = { field: ['host.name', 'source.ip'], value: 2 };
+    it('Generates aggregation without cardinality', async () => {
+      const aggs = buildThresholdMultiBucketAggregation({
+        threshold,
+        timestampField,
+        sortKeys: undefined,
+      });
+      expect(aggs).toStrictEqual({
+        thresholdTerms: {
+          aggs: {
+            count_check: {
+              bucket_selector: {
+                buckets_path: {
+                  docCount: '_count',
+                },
+                script: 'params.docCount >= 2',
+              },
+            },
+            max_timestamp: {
+              max: {
+                field: '@timestamp',
+              },
+            },
+            min_timestamp: {
+              min: {
+                field: '@timestamp',
+              },
+            },
+          },
+          composite: {
+            after: undefined,
```

Comment:
The

```
after: undefined
```

in this test and the one below would  become:

```ts
after: null,
```

(to be valid JSON) if [this suggestion](https://github.com/elastic/kibana/pull/131985/files#r924737563) is implemented